Tests/SwiftParserTest/Declarations.swift

```diff
@@ -612,8 +611,7 @@ final class DeclarationTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(locationMarker: "MISSING_COLON", message: "Expected ':' in function parameter"),
         DiagnosticSpec(locationMarker: "MISSING_RPAREN", message: "Expected ')' to end parameter clause"),
-        // FIXME: We should issues something like "Expected identifier in declaration"
-        DiagnosticSpec(locationMarker: "MISSING_IDENTIFIER", message: "Expected '' in struct"),
+        DiagnosticSpec(locationMarker: "MISSING_IDENTIFIER", message: "Expected identifier in struct"),
```

Comment:
"in struct" sounds a little weird to me, but the change to give them names looks good to me 👍

Reply:
This will be changed to `Expected name of struct` in one of my future PRs. I can’t find it right now, so it looks like I haven’t put it up yet.